doc/rst/developer/chips/10.rst

```diff
@@ -527,7 +525,7 @@ For more details on when the copy initializer would be called, please refer to
 `CHIP 13 - When Do Records and Array Copies Occur`_
 
 .. _CHIP 13 - When Do Records and Array Copies Occur:
-   https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/chips/13.rst
+   https://github.com/chapel-lang/chapel/blob/master/doc/developer/chips/13.rst
```

Comment:
@noakesmichael, you're going to want to change that back - I didn't make that update and without it the link is broken